how-to-guides/create-delete-projects/notebooks/Create_delete_projects.ipynb

```diff
@@ -60,7 +60,11 @@
     ]
    },
    "source": [
-    "Install `neptune-client`"
+    "# Before you start\n",
+    "\n",
+    "Make sure that you have:\n",
+    "* [Python 3.7+ installed](https://www.python.org/downloads/),\n",
```

Comment:
I would not list these as hard prerequisites, since many users will run this in Colab

Reply:
This "Before you start" for this notebook will be a bit different from the template as we need the users to be registered to be able to create/delete projects.